app/js/components/orderEntry/ActiveOrders.jsx

```diff
@@ -2,15 +2,24 @@ import React from 'react';
 import { connect } from 'react-redux';
 import format from 'date-fns/format';
 import PropTypes from 'prop-types';
-import activeOrderAction from '../../actions/activeOrderAction';
+import ReactPaginate from 'react-paginate';
+import { activeOrderAction } from '../../actions/activeOrderAction';
 import { addDraftOrder } from '../../actions/draftTableAction';
 import imageLoader from '../../../img/loading.gif';
 
-export class ActiveOrders extends React.Component {
-  state = {
-    draftOrder: {},
-  };
 
+export class ActiveOrders extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      draftOrder: {},
+      pageCount: 0,
+      limit: 1,
```

Comment:
You have a page limit of 1 active order per page, is there a reason for the choice of only 1?

Reply:
Was using it for testing. Have changed it. Thank u @fuchodeveloper